node.go

```diff
@@ -49,6 +49,7 @@ func setNodeInterface(name string) bool {
 	// does not specify a specific interface generate a random Node ID
 	// (section 4.1.6)
 	if name == "" {
+		ifname = "random"
```

Comment:
I would tend to use "erewhon" for historical reasons :-)

I understand your use of the name "random", but wonder if "none" would be more clear.